src/clr-angular/popover/common/abstract-popover.spec.ts

```diff
@@ -15,6 +15,63 @@ import { ClrPopoverToggleService } from '../../utils/popover/providers/popover-t
 import { AbstractPopover } from './abstract-popover';
 import { POPOVER_HOST_ANCHOR } from './popover-host-anchor.token';
 
+@Component({
```

Comment:
To get TS working with TS projects and storybook the tsconfigs needed to be updated to run the tests as es2015. This speeds up the tests as a side effect but forced this code to be reordered since true ES2015 classes are not hoisted like the compiled ES5 equivalent.